### PR TITLE
Added documentation links as comments

### PR DESCRIPTION
**Pull request by neptunel**
Added related documentation links for each listener in the starter template.

### CODE REVIEW

File: listeners/messages/sample-message.js [hunk 1]
```diff
@@ -1,3 +1,10 @@
+/*
+   Learn how to:
+   -> listen messages on Bolt:  https://slack.dev/bolt-js/concepts#message-listening
+   -> send messages on Bolt: https://slack.dev/bolt-js/concepts#message-sending
```
Comment (srajiang) [inline listeners/messages/sample-message.js:4]:
```suggestion
   -> send messages with Bolt: https://slack.dev/bolt-js/concepts#message-sending
```

File: listeners/events/app-home-opened.js [hunk 2]
```diff
@@ -1,3 +1,10 @@
+/*
+
+   Learn how to:
+   -> use events on Bolt:  https://slack.dev/bolt-js/concepts#event-listening
```
Comment (stevengill) [inline listeners/events/app-home-opened.js:4]:
```suggestion
   -> use events with Bolt:  https://slack.dev/bolt-js/concepts#event-listening
```

**Review by stevengill — CHANGES_REQUESTED** [review]
Looks great!

**Review by neptunel — COMMENTED** [review]
Fixed the grammar tweaks! Can I merge now @stevengill ?

**Review by stevengill — APPROVED** [review]

**Pull request merged**